deployments/salt/splunk-otel-collector/fluentd_config.sls

```diff
@@ -0,0 +1,17 @@
+{% set splunk_fluentd_config = salt['pillar.get']('splunk-otel-collector:splunk_fluentd_config', '/etc/otel/collector/fluentd/fluent.conf') %}
+
+{% set splunk_fluentd_config_source = salt['pillar.get']('splunk-otel-collector:splunk_fluentd_config_source') %}
+
+{% if splunk_fluentd_config_source != '' %}
+
+Push custom FluentD config, if provided:
+  file.managed:
```

Comment:
Please make sure that the td-agent service is restarted if the fluentd config has changed.

Reply:
We have added fluentd_service.sls file for restarting td-agent service if the fluentd config file has changed.